src/_sass/gnome-shell/components/gdm/_screen-shield.scss

```diff
@@ -11,73 +9,83 @@ $_screenshield_shadow: 0px 0px 6px rgba(0, 0, 0, 0.726);
   width: 80px;
   height: 48px;
   -arrow-thickness: 12px;
-  -arrow-shadow: $_screenshield_shadow;
+  -arrow-shadow: $shadow-1;
 }
 
 .screen-shield-clock {
   color: white;
-  text-shadow: $_screenshield_shadow;
-  font-weight: bold;
+  text-shadow: $shadow-1;
+  font-weight: normal;
   text-align: center;
   padding-bottom: 1.5em;
 }
 
 .screen-shield-clock-time {
```

Comment:
.screen-shield-* suffix appears to have been changed to .unlock-dialog-* in the upstream

https://gitlab.gnome.org/GNOME/gnome-shell-sass/-/blob/master/widgets/_screen-shield.scss

Reply:
Fixed.